lib/slack-rtmapi.rb

```diff
@@ -17,4 +23,39 @@ def self.get_url(options)
     URI(body['url'])
   end
 
+  def initialize(options)
+    @token = options.fetch(:token)
```

Comment:
fetch will give a clear error message if this required parameter is not passed, it is much terser than the if - raise - else construct